pkg/features/kube_features.go

```diff
@@ -743,9 +743,9 @@ const (
 	// owner: @derekwaynecarr
 	// alpha: v1.20
 	// beta: v1.22
-	//
+	// GA: v1.32
 	// Enables kubelet support to size memory backed volumes
-	SizeMemoryBackedVolumes featuregate.Feature = "SizeMemoryBackedVolumes"
+	SizeMemoryBackedVolumes featuregate.Feature = "SizeMemoryBackedVolumes" // remove in 1.35
```

Comment:
can you please add a comment saying that this FG is only used in kubelet and not needed for emulated version so in couple releases it will be easier to understand that we can just remove it

Reply:
I will do this tonight or tomorrow.

Reply:
done.